app/src/main/java/de/westnordost/streetcomplete/quests/powerpoles_material/AddPowerPolesMaterial.kt

```diff
@@ -20,6 +23,15 @@ class AddPowerPolesMaterial : OsmFilterQuestType<PowerPolesMaterial>() {
 
     override fun getTitle(tags: Map<String, String>) = R.string.quest_powerPolesMaterial_title
 
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("nodes with power = pole or man_made = utility_pole") +
+        // and also show the (power) lines themselves
+        getMapData().filter("ways with power ~ line|minor_line or communication = line or telecom = line")
```

Comment:
This loops the map data twice, but I guess `nodes with … or ways with …` is not valid ElementFilterExpression syntax?